transaction_integration_test.go

```diff
@@ -54,6 +54,25 @@ func TestTransactionCreateSubmitForSettlementAndVoid(t *testing.T) {
 		t.Fatalf("transaction settlement amount (%s) did not equal amount requested (%s)", amount, ten)
 	}
 
+	// Clone
+	tx4, err := testGateway.Transaction().Clone(tx2.Id, &TransactionCloneRequest{
+		Amount: ten,
+		Options: &TransactionOptions{
+			SubmitForSettlement: true,
+		},
+	})
+	t.Log(tx4)
+
+	if err != nil {
+		t.Fatal(err)
+	}
+	if x := tx4.Status; x != TransactionStatusSubmittedForSettlement {
+		t.Fatal(x)
+	}
+	if amount := tx4.Amount; amount.Cmp(ten) != 0 {
+		t.Fatalf("transaction settlement amount (%s) did not equal amount requested (%s)", amount, ten)
+	}
+
```

Comment:
This test is focusing purely on submitting for settlement and then voiding. Can you move the expectations you've added here into a new test that is focused on creating a transaction and cloning it?